### PR TITLE
Move config files to follow XDG spec

### PR DESCRIPTION
**Pull request by rbvigilante**
Fixes https://github.com/redbubble/yak/issues/28

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -79,28 +79,61 @@ func init() {
...
-		os.Exit(1)
+	if configPath == "" {
+		home, err := homedir.Dir()
+
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+
+		configPath = path.Join(home, ".lonfig")
```
Comment (toothbrush) [inline cmd/root.go]: Typo!  Not `.lonfig`!

**Comment by toothbrush** [comment]
This is fine.  I'm ambivalent but the one good argument i've heard so far is that `$XDG_HOME` allows users to put the config wherever they choose, as opposed to us foisting our choices upon them.

File: cmd/root.go (hunk 1 above)
Comment (toothbrush) [inline cmd/root.go]: Typo!  Not `.lonfig`!

**Comment by rb-johnmurphy** [comment]
It would be good to support both, given it's already been released without XDG and people are using it.

**Comment by rbvigilante** [comment]
Now the old and the new way are both supported. Better @rb-johnmurphy?

File: README.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ yak --list-roles
 
 ### Configuring
 
-Yak can be configured with a configuration file at  `~/.yak/config.toml`.
+Yak can be configured with a configuration file at  `~/.config/yak/config.toml` (`~/.yak/config.toml` is also supported).
```
Comment (toothbrush) [inline README.md]: I'm not sure if i'd mention the legacy thing.  Just complicates the issue, no?

Comment (rbvigilante) [reply README.md]: I think having it there and undocumented is worse, but that's just an opinion.

**Pull request merged**